api/client/client_test.go

```diff
@@ -0,0 +1,164 @@
+package client
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"net"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/gravitational/teleport/api/client/proto"
+
+	"github.com/gravitational/trace"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/crypto/ssh"
+	"google.golang.org/grpc"
+)
+
+var (
+	authAddr = "localhost:3025"
+	server   = newMockServer()
+)
```

Comment:
These should be probably moved inside a setup function to be called from the test functions.